src/sentry/static/sentry/app/views/onboarding/createProject.jsx

```diff
@@ -135,70 +117,144 @@ const CreateProject = createReactClass({
+              <ProjectNameInput>
+                <ProjectPlatformicon monoTone platform={platform} />
+                <input
+                  type="text"
+                  name="name"
+                  label={t('Project Name')}
+                  placeholder={t('Project name')}
+                  autoComplete="off"
+                  value={projectName}
+                  onChange={e => this.setState({projectName: e.target.value})}
+                />
+              </ProjectNameInput>
+            </div>
+            <div>
+              <FormLabel>{t('Assign a Team')}</FormLabel>
+              <TeamSelectInput>
+                <SelectControl
+                  name="select-team"
+                  clearable={false}
+                  value={team}
+                  placeholder={t('Select a Team')}
+                  onChange={val => this.setState({team: val})}
+                  options={teams.map(({slug}) => ({
+                    label: `#${slug}`,
+                    value: slug,
+                  }))}
+                />
+                <Tooltip title={t('Create a team')}>
                   <Button
-                    className="ref-create-team"
-                    priority="primary"
+                    borderless
+                    data-test-id="create-team"
+                    type="button"
+                    icon="icon-circle-add"
                     onClick={() =>
                       openCreateTeamModal({
                         organization,
+                        onClose: ({slug}) => this.setState({team: slug}),
                       })}
-                  >
-                    {t('Create a Team')}
-                  </Button>
-                </CreateTeamBody>
-              </React.Fragment>
-            }
-          />
-        )}
-      </div>
+                  />
+                </Tooltip>
+              </TeamSelectInput>
+            </div>
+            <div>
+              <Button
+                data-test-id="create-project"
+                priority="primary"
+                disabled={inFlight || !team || projectName === ''}
+              >
+                {t('Create Project')}
+              </Button>
+            </div>
+          </CreateProjectForm>
+        </div>
+      </React.Fragment>
     );
-  },
-});
+  }
+}
+
+export default withApi(withTeams(withOrganization(CreateProject)));
+export {CreateProject};
+
+const CreateProjectForm = styled('form')`
+  display: grid;
+  grid-template-columns: 300px 250px max-content;
+  grid-gap: ${space(2)};
+  align-items: end;
+  padding: ${space(3)} 0;
+  margin-top: ${space(2)};
+  box-shadow: 0 -1px 0 rgba(0, 0, 0, 0.1);
+  background: #fff;
+  position: sticky;
+  bottom: 0;
+`;
 
-export default CreateProject;
+const FormLabel = styled('div')`
+  font-size: 18px;
+  margin-bottom: ${space(1)};
+`;
+
+const ProjectPlatformicon = styled(PlatformiconTile)`
+  font-size: 25px;
```

Comment:
`p.theme.headerFontSize` unless you think 25px is a lot better.

Reply:
This is specifically for adjusting the icon size, so it's not actually for text.